docker/cli
# Add hyperlink to Formatting

**- What I did**
Change the text of `https://docs.docker.com/go/formatting/` to a hyperlink.

**- How I did it**
Change the text to Markdown hyperlink format.

**- How to verify it**
Make sure you see the text as a link and that it's clickable and takes you to the desired page.

**- Description for the changelog**

Change the text of `https://docs.docker.com/go/formatting/` to a hyperlink.

**- A picture of a cute animal (not mandatory but encouraged)**
